### ISSUE TITLE
Installation on Mac

### ISSUE THREAD

**Issue by lschult2** [arpruss/raspberryjammod#44]
I'm working on installing on a Mac (MacOS Mojave), and am having issues.  I copied the files listed in README (into mods/ and mcpipy/), but that does not appear to be sufficient.  Did I miss something?

I also tried running "sh build.sh", but get a syntax error on line 4.

I'm using the latest version of Minecraft and Forge: 1.14.3.  It is a version compatibility issue?  Does this only work on Minecraft 1.12.2 and lower?

**Comment by perfectlyGoodInk** [comment]
Yes. As far as I understand, it currently only works on 1.12.2 and lower.

**Comment by Marcinosoft** [comment]
If you want to install mod for python coding or run samples - all you need to do is unpack:
https://github.com/arpruss/raspberryjammod/blob/master/mods.zip
into Minecraft mods folder (I assume that Minecraft and Forge are already installed). Zip contains RaspberryJamMod.jar for different MC versions, but up to 1.12.2. Forge will load jar from folder with version equal to MC version you'll launch.
Next copy
https://github.com/arpruss/raspberryjammod/tree/master/mcpipy
folder where you want to place python source codes. It means anywhere. From this folder you can run samples.

But if you want to develop code of this plugin in java, then build.sh is shell script to compile project into jar files for different MC versions, currently up to 1.12.2. If you have syntax error then probably your default shell (sh) is not able to handle do-done syntax. There is no shebang in first line, so you may add it by your own as very first line, specyfying for example bash:
#!/bin/bash
or run "bash" before running script or execute it like this:
bash build.sh
